src/main/java/org/sagebionetworks/bridge/services/Exporter3Service.java

```diff
@@ -61,10 +69,82 @@
+        createdOnColumn.setName("createdOn");
+        createdOnColumn.setColumnType(ColumnType.DATE);
+        listBuilder.add(createdOnColumn);
+
+        ColumnModel modifiedOnColumn = new ColumnModel();
+        modifiedOnColumn.setName("modifiedOn");
+        modifiedOnColumn.setColumnType(ColumnType.DATE);
+        listBuilder.add(modifiedOnColumn);
+
+        // We have pre-existing validation that checks that the total length of all data groups is less than
+        // 250 characters. This validation predates the use of string lists in Synapse. Additionally, some apps have
+        // 30+ data groups, and some apps have data groups with 20+ characters. Synapse table row width is valuable, so
+        // instead of reserving potentially 600+ characters that will be sparsely used, we will keep to the
+        // 250 character limit.
+        ColumnModel dataGroupsColumn = new ColumnModel();
+        dataGroupsColumn.setName("dataGroups");
+        dataGroupsColumn.setColumnType(ColumnType.STRING);
+        dataGroupsColumn.setMaximumSize(250L);
+        listBuilder.add(dataGroupsColumn);
+
+        // These are 2-character language codes (eg "en", "es"). Let's generously allow participants to export up to
+        // 10 languages, which is 20 characters at most, and should be more than enough for nearly all participants.
+        ColumnModel languagesColumn = new ColumnModel();
+        languagesColumn.setName("languages");
+        languagesColumn.setColumnType(ColumnType.STRING_LIST);
+        languagesColumn.setMaximumSize(2L);
+        languagesColumn.setMaximumListLength(10L);
+        listBuilder.add(languagesColumn);
+
+        // The longest sharing scope is ALL_QUALIFIED_RESEARCHERS, which is 25 characters long.
+        ColumnModel sharingScopeColumn = new ColumnModel();
+        sharingScopeColumn.setName("sharingScope");
+        sharingScopeColumn.setColumnType(ColumnType.STRING);
+        sharingScopeColumn.setMaximumSize(25L);
+        listBuilder.add(sharingScopeColumn);
+
+        // Synapse doesn't have maps as a type in their Table model. For now, we serialize it as a string. In 2.0, we
+        // set a max string length of 250 characters, and that seems to be fine.
+        ColumnModel studyMembershipsColumn = new ColumnModel();
+        studyMembershipsColumn.setName("studyMemberships");
+        studyMembershipsColumn.setColumnType(ColumnType.STRING);
+        studyMembershipsColumn.setMaximumSize(250L);
+        listBuilder.add(studyMembershipsColumn);
+
+        // The longest time zone I could find was 28 characters long. Let's add a bit of a buffer and say 40
+        // characters. As long as Llanfair PG, Wales doesn't have its own time zone, this should be fine.
+        ColumnModel timeZoneColumn = new ColumnModel();
+        timeZoneColumn.setName("timeZone");
+        timeZoneColumn.setColumnType(ColumnType.STRING);
+        timeZoneColumn.setMaximumSize(40L);
+        listBuilder.add(timeZoneColumn);
```

Comment:
We call this client time zone, vs. study time zone which is the time zone in which the study is being run or in which they coordinators want to sees times.

Reply:
Renamed to clientTimeZone.